RAML/ez.raml

```diff
@@ -0,0 +1,1277 @@
+                If-match:
+                    description: Causes to patch only if the specified etag is the current one
+                Content-Type:
+                    description: application/vnd.ez.api.VersionUpdate+json or application/vnd.ez.api.VersionUpdate+xml
+            body:
+              application/vnd.ez.api.Version+xml:
+                  schema:
+                  example:  |
+                      <?xml version="1.0" encoding="UTF-8"?>
+                      <VersionUpdate xmlns:p="http://ez.no/API/Values"
+                        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                        xsi:schemaLocation="http://ez.no/API/Values ../VersionUpdate.xsd ">
+                        <modificationDate>2001-12-31T12:00:00</modificationDate>
+                        <initialLanguageCode>ger-DE</initialLanguageCode>
+                        <fields>
+                          <field>
+                            <id>1234</id>
+                            <fieldDefinitionIdentifier>title</fieldDefinitionIdentifier>
+                            <languageCode>ger-DE</languageCode>
+                            <fieldValue>Neuer Titel</fieldValue>
+                          </field>
+                          <field>
+                            <id>1235</id>
+                            <fieldDefinitionIdentifier>summary</fieldDefinitionIdentifier>
+                            <languageCode>ger-DE</languageCode>
+                            <fieldValue>Dies ist eine neue Zusammenfassungy</fieldValue>
+                          </field>
+                        </fields>
+                      </VersionUpdate>
+            responses:
+                200:
+                    body:
+                        application/vnd.ez.api.Version+xml:
+                            example:  |
+                                <?xml version="1.0" encoding="UTF-8"?>
+                                <Version href="/content/objects/23/versions/4" media-type="application/vnd.ez.api.Version+xml">
+                                  <VersionInfo>
+                                    <id>45</id>
+                                    <versionNo>4</versionNo>
+                                    <status>DRAFT</status>
+                                    <modificationDate>2012-02-20T12:00:00</modificationDate>
+                                    <Creator href="/user/users/44" media-type="application/vnd.ez.api.User+xml" />
+                                    <creationDate>22012-02-20T12:00:00</creationDate>
+                                    <initialLanguageCode>ger-DE</initialLanguageCode>
+                                    <names>
+                                      <value languageCode="ger-DE">Neuer Titel</value>
+                                    </names>
+                                    <Content href="/content/objects
```

Comment:
Nitpick: missing end of line, it might be important to some text parsers, that's why GitHub marks it in a specific way.